TROUBLESHOOTING.md

```diff
@@ -58,6 +58,10 @@ report sent to us if you have it.
 
 You can enable the PyTorch/XLA debugging tool by setting `PT_XLA_DEBUG=1`, which provides a couple useful debugging features.
 
+## PyTorch/XLA + Dynamo Debugging Tool
+
+You can enable the PyTorch/XLA + Dynamo debugging tool by setting `XLA_DYNAMO_DEBUG=1`, which provides a couple useful debugging features.
```

Comment:
(Nit-picking)
I see that this is copied from above, but without mentioning those "couple [of] useful debugging features", the statement is redundant.
I suggest we drop everything after the comma (`, which provides...`).

Reply:
Oh I can add a suggestion, let me do that.